cloudant.js

```diff
@@ -226,6 +200,7 @@ function Cloudant(options, callback) {
   };
 
   // add top-level Cloudant-specific functions
+  nano.auth = undefined; // handled by cookie plugin
```

Comment:
Yeah, I guess I could leave this in. I was inclined to remove the nano authentication layer entirely. Perhaps I need to revisit the `ping` implementation too then. If the cookie plugin is used then ping is just a HEAD request, otherwise it can be the legacy fetch session thing. Also, I can put all those horrid tests back. Yay!

Reply:
I guess it's really a backward compatibility thing. If ping/auth worked as per nano before (without the cookie plugin) then they need to continue to do so.